packages/components/src/components/modal/modal.tsx

```diff
@@ -146,7 +144,6 @@ const ModalElement = ({
             style={{
                 height: height || 'auto',
                 width: width || 'auto',
-                minHeight: min_height || 0,
```

Comment:
minHeight was being overridden to zero if we don't provide minHeight to Modal, breaking existing implementations of modal where min height was already set through css class